backend/inspirehep/search/mappings/v7/records/authors.json

```diff
@@ -274,7 +277,8 @@
             "copy_to": [
               "authorautocomplete",
               "names_analyzed_initials",
-              "names_analyzed"
+              "names_analyzed",
+              "name_keyword"
```

Comment:
I don't think that's a correct use of `copy_to`. It's meant when you want to combine several fields into one. Better define a `value.raw` of type `keyword` as a variant, like we do for example here: https://github.com/inspirehep/inspirehep/blob/95af58d4b474a36dc9b0a1f37560d831fe793106/backend/inspirehep/search/mappings/v7/records/hep.json#L200-L204

Reply:
I guess it's also better to have `lowercase_normalizer` just incase?